docs/detections/rules-ui-create.asciidoc

```diff
@@ -289,7 +289,9 @@ TIP: Click the help icon (image:images/esql-ref-button.png[Click the ES|QL help
 [[esql-agg-query]]
 ===== Aggregating query 
 
-Aggregating queries use {ref}/esql-functions-operators.html#esql-agg-functions[`STATS...BY`] functions to aggregate source event data. Alerts generated by an {esql} rule with an aggregating query only contain the fields returned by the query.
+Aggregating queries use {ref}/esql-functions-operators.html#esql-agg-functions[`STATS...BY`] functions to aggregate source event data. Alerts generated by a rule with an aggregating query only contain the source event fields that the {esql} query returns and new fields that the query creates.
```

Comment:
@vitaliidm based on your feedback [here](https://github.com/elastic/security-docs/pull/5182#discussion_r1600051394), I just want to double-check that the description of fields returned for aggregating queries is correct.

Reply:
Technically, it's correct. Since most of fields any query returns are form source event.
But using source event fields might be too close to the case of non-aggr query and might lead to misunderstandings

I would try to simplify

> Aggregating queries use {ref}/esql-functions-operators.html#esql-agg-functions[`STATS...BY`] functions to aggregate source event data. Alerts generated by an {esql} rule with an aggregating query only contain the fields returned by the query, including new fields

Reply:
```suggestion
Aggregating queries use {ref}/esql-functions-operators.html#esql-agg-functions[`STATS...BY`] functions to aggregate source event data. Alerts generated by a rule with an aggregating query only contain the fields that the {esql} query returns and new fields that the query creates.
```